src/App.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .App-header {
-  min-height: 100vh;
+  /* min-height: 100vh; */
```

Comment:
please remove your code, if you don't want it.

Reply:
Please make your page more simple. Here is a UI demo.  This is already enough.
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/10722490/185362613-bb952c31-da93-41e5-87a9-3d8866a76d92.png">